.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
```

Comment:
This check makes no sense outside of a pull_request event

Reply:
Changes done, and synced with https://github.com/osrd-project/osrd/pull/6528 in https://github.com/osrd-project/osrd-website/pull/171/commits/1026be44ef3e743fa5228950df208743b768d018